### ISSUE TITLE
Konsole URL hints broken

### ISSUE THREAD

**Issue by octoploid** [tmux/tmux#845]
Konsole URL hints are broken on latest trunk. They worked fine a few weeks ago.
So somehow the tmux character rendering throws the konsole URL recognition code off the tracks.
The end result are invalid URLs that contain random surrounding text.

**Comment by nicm** [comment]
But they look alright on the terminal, right? This seems like a problem
with Konsole not with tmux.


On 30 Mar 2017 8:51 am, "octoploid" <notifications@github.com> wrote:

Konsole URL hints are broken on latest trunk. They worked fine a few weeks
ago.
So somehow the tmux character rendering throws the konsole URL recognition
code off the tracks.
The end result are invalid URLs that contain random surrounding text.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tmux/tmux/issues/845>, or mute the thread

**Comment by octoploid** [comment]
Well, as I wrote, it worked fine with an older tmux.
Yes, the URLs themselves are rendered OK, but I cannot use them anymore, by clicking
or by using a keyboard shortcut, because the links contain garbage.

**Comment by nicm** [comment]
Right but we changed how tmux draws things, we can't be responsible for how
Konsole detects URLs, there is no information about URLs in the output from
tmux. It is Konsole's algorithm to detect them that is going wrong, not
tmux's.


On 30 Mar 2017 9:49 am, "octoploid" <notifications@github.com> wrote:

Well, as I wrote, it worked fine with an older tmux.
Yes, the URLs themselves are rendered OK, but I cannot use them anymore, by
clicking
or by using a keyboard shortcut, because the links contain garbage.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/tmux/tmux/issues/845#issuecomment-290330608>, or mute
the thread
